### DISCUSSION TITLE
Problem with template and x-teleport

### DISCUSSION THREAD

**Discussion by jesusramiropr** [alpinejs/alpine#4461] [7. Bugs (archived)] [answered]
From version 3.14.4 and 3.14.5, using x-teleport="body" the events and code of Alpine (@click, x-text, x-if, etc) dont' work.

It is like x-ignore, but before that versions, the template with x-teleport works fine.

Has the functionality changed or is this a bug?

**Comment by ekwoka** [comment]
This is likely a regression. Does it still not work on 3.14.6?

can you make an example on https://awesomealpine.com/play?

**Reply by jesusramiropr** [reply]
https://awesomealpine.com/play?coreplugins=0&html=PGJ1dHRvbgogIHR5cGU9ImJ1dHRvbiIKICBAY2xpY2s9IiRzdG9yZS50b2FzdC5zaG93KHsgdHlwZTogJ2Vycm9yJywgJ3RleHQnOiAnSGVyZSBnb2VzIHRoZSBzdWNjZXNzIHRleHQnIH0pIgogIHgtZGF0YQo-CiAgT3BlbiB0b2FzdAo8L2J1dHRvbj4KCjx0ZW1wbGF0ZQogIHgtdGVsZXBvcnQ9ImJvZHkiCiAgeC1kYXRhCj4KICA8ZGl2CiAgICBjbGFzcz0iZ3JpZCBncmlkLWNvbHMtMSBnYXAtNCB3LWZ1bGwgbWF4LXctbGcgcHgtZ3V0dGVyIG1kOnB4LWd1dHRlci10YWJsZXQgZml4ZWQgdG9wLTEwIGxlZnQtMS8yIHRyYW5zZm9ybS1ncHUgLXRyYW5zbGF0ZS14LTEvMiB6LVsyMDBdIgogICAgeC1zaG93PSIkc3RvcmUudG9hc3QuaXRlbXMubGVuZ3RoIgogID4KICAgIDx0ZW1wbGF0ZQogICAgICB4LWZvcj0ibWVzc2FnZSBpbiAkc3RvcmUudG9hc3QuaXRlbXMiCiAgICAgIDprZXk9Im1lc3NhZ2UuaWQiCiAgICA-CiAgICAgIDxkaXYKICAgICAgICBjbGFzcz0iZmxleCBpdGVtcy1jZW50ZXIgZ2FwLTMgcHgtNCBweS0yIHJvdW5kZWQtbWQgYm9yZGVyIGJvcmRlci10cmFuc3BhcmVudCIKICAgICAgICA6Y2xhc3M9InsgJ2JnLWJsdWUtMTAwJzogbWVzc2FnZS50eXBlID09PSAnaW5mbycsICdiZy1ncmVlbi0xMDAnOiBtZXNzYWdlLnR5cGUgPT09ICdzdWNjZXNzJywgJ2JnLXJlZC0xMDAnOiBtZXNzYWdlLnR5cGUgPT09ICdlcnJvcicsICdiZy15ZWxsb3ctMTAwJzogbWVzc2FnZS50eXBlID09PSAnd2FybmluZycsIH0iCiAgICAgID4KICAgICAgICA8dGVtcGxhdGUgeC1pZj0ibWVzc2FnZS50eXBlID09PSAnaW5mbyciPgogICAgICAgICAgPHNwYW4-aWNvbm8gaW5mbzwvc3Bhbj4KICAgICAgICA8L3RlbXBsYXRlPgogICAgICAgIDx0ZW1wbGF0ZSB4LWlmPSJtZXNzYWdlLnR5cGUgPT09ICdzdWNjZXNzJyI-CiAgICAgICAgICA8c3Bhbj5pY29ubyBzdWNjZXNzPC9zcGFuPgogICAgICAgIDwvdGVtcGxhdGU-CiAgICAgICAgPHRlbXBsYXRlIHgtaWY9Im1lc3NhZ2UudHlwZSA9PT0gJ2Vycm9yJyI-CiAgICAgICAgICA8c3Bhbj5pY29ubyBlcnJvcjwvc3Bhbj4KICAgICAgICA8L3RlbXBsYXRlPgogICAgICAgIDx0ZW1wbGF0ZSB4LWlmPSJtZXNzYWdlLnR5cGUgPT09ICd3YXJuaW5nJyI-CiAgICAgICAgICA8c3Bhbj5pY29ubyB3YXJuaW5nPC9zcGFuPgogICAgICAgIDwvdGVtcGxhdGU-CiAgICAgICAgPGRpdgogICAgICAgICAgY2xhc3M9Im1lc3NhZ2VfX3RleHQiCiAgICAgICAgICB4LWh0bWw9Im1lc3NhZ2UudGV4dCIKICAgICAgICA-PC9kaXY-CiAgICAgICAgPGJ1dHRvbgogICAgICAgICAgQGNsaWNrPSIkc3RvcmUudG9hc3QucmVtb3ZlKG1lc3NhZ2UuaWQpIgogICAgICAgICAgdHlwZT0iYnV0dG9uIgogICAgICAgID4KICAgICAgICAgIENsb3NlIHRvYXN0CiAgICAgICAgPC9idXR0b24-CiAgICAgIDwvZGl2PgogICAgPC90ZW1wbGF0ZT4KICA8L2Rpdj4KPC90ZW1wbGF0ZT4K&script=QWxwaW5lLnN0b3JlKCd0b2FzdCcsIHsKICBpdGVtczogW10sCiAgZGVmYXVsdERlbGF5OiA3MDAwLAogIHNob3coaXRlbSkgewogICAgaXRlbS5pZCA9IERhdGUubm93KCk7CiAgICB0aGlzLml0ZW1zLnVuc2hpZnQoaXRlbSk7CiAgICB0aGlzLnNjaGVkdWxlUmVtb3ZhbChpdGVtLmlkLCBpdGVtLmRlbGF5KTsKICB9LAogIHNjaGVkdWxlUmVtb3ZhbChpZCwgZGVsYXkgPSB0aGlzLmRlZmF1bHREZWxheSkgewogICAgaWYgKGRlbGF5KSB7CiAgICAgIHNldFRpbWVvdXQoKCkgPT4gdGhpcy5yZW1vdmUoaWQpLCBkZWxheSk7CiAgICB9CiAgfSwKICByZW1vdmUoaWQpIHsKICAgIGNvbnN0IGl0ZW1JbmRleCA9IHRoaXMuaXRlbXMuZmluZEluZGV4KAogICAgICAoaXRlbSkgPT4gaXRlbS5pZCA9PT0gaWQsCiAgICApOwogICAgdGhpcy5pdGVtcy5zcGxpY2UoaXRlbUluZGV4LCAxKTsKICB9LAp9KTsK&tw=1

3.14.4 - 3.14.5 - 3.14.16 still does not work

**Reply by ekwoka** [reply]
Awesome thanks. (there was a share button btw for a smaller url lol)

I'll be getting some time into it later.

**Reply by joshhanley** [reply]
I've replied on https://github.com/alpinejs/alpine/discussions/4467#discussioncomment-11479445 about this issue.

Copied message below

> I had a look this with Caleb on our call this morning and we think the teleport `clone._x_ignore = true ` was something for Livewire. But we're not 100% on that. Caleb said to just try removing it and see if both test suites pass.
> 
> So I've submitted an Alpine PR #4469 with it removed and a Livewire PR https://github.com/livewire/livewire/pull/9072 with the Alpine fix and both test suites are passing.

**Comment by joshhanley** [comment, marked as answer]
This has been fixed in v3.14.7.
